src/page/interactive_bom.jsx

```diff
@@ -0,0 +1,35 @@
+const React = require('react')
+const semantic = require('semantic-ui-react')
+const ReactResponsive = require('react-responsive')
+
+const {folder} = require('../zip-info.json')
+const ibomUrl = `https://kitspace.org/${folder}/interactive_bom.html`
+
+let InteractiveBOM = React.createClass({
+  render() {
+    return (
+      <ReactResponsive
+        query={
+        '(max-width: 920px)' // XXX if you change this change it in the .scss too!
+        }
+      >
```

Comment:
Avoid `ReactResponsive` if you can. It causes jumps in layout and can make the sight feel sluggish. I just main managed to get it out of the title bar. I'd like to get rid of it alltogether.

Reply:
Will do. (See "clueless about React" comment above...)